.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Run all tests with Maven
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
change this to all branches - the tests should run everytime you push to your feature branch